package.json

```diff
@@ -29,6 +31,7 @@
   "homepage": "https://github.com/pelias/dbclient",
   "devDependencies": {
     "intercept-stdout": "^0.1.2",
+    "npm-check": "git://github.com/orangejulius/npm-check.git#disable-update-check",
```

Comment:
Is it possible that this line could cause a problem?  When I leave it in I get a lot of packages that fail to install using NPM install.  If I remove it, the package seems to install as expected.